lm_eval/tasks/squadv2/with_noans_prob.yaml

```diff
@@ -1,4 +1,4 @@
 group: squadv2_complete
 task:
-  - squadv2
+  - squadv2_generate_until
```

Comment:
we want to drop the YAML versions of SQUADv2 now that the old one is supported, no?